### PR TITLE
fix/update event namespaces for succession events

### PR DESCRIPTION
**Pull request by jkoenig134**

### CODE REVIEW

File: packages/runtime/package.json
Comment (Milena-Czierlinski) [inline packages/runtime/package.json:1]: What was our policy on bumping the runtime if we bump an underlying lib/workspace?

Comment (jkoenig134) [reply packages/runtime/package.json:1]: I don't get your question. I want this update to ship so I also updated the runtime.

Comment (Milena-Czierlinski) [reply packages/runtime/package.json:1]: I had in mind, that we bump the runtime version itself, too, if an underlying lib is bumped. Or do we only do that in special cases?

Comment (jkoenig134) [reply packages/runtime/package.json:1]: lol, I was TOTALLY sure I bumped the runtime :D that's where my confusion came from.

**Pull request merged**